app/services/registration/referrals.py

```diff
@@ -29,25 +30,44 @@ def __init__(
         self._lmr_encryption_service = lmr_encryption_service
 
     def register(self, bsn: BSN, data_domain: str) -> Referral | None:
-        pseudonym = self.pseudonym_service.submit(PseudonymCreateDto(bsn=bsn))
+        recipient_organization = "ura:" + self._ura_number
```

Comment:
This should be configurable.., this is the organisation where the NVI runs..

Something like nvi configuration ...